repose-aggregator/docs/src/asciibinder/welcome/ver-9-upgrade-notes.adoc

```diff
@@ -76,6 +76,8 @@
     }
 ]]></scripting>
 ----
+. The <<../architecture/filter-chain.adoc#, Filter Chain>> now determines whether or not a filter should run based on the current state of the request rather than the state of the request when it was initially received.
+  For more details, see <<../architecture/filter-chain.adoc#filter-activation-determination, filter activation determination>>.
```

Comment:
Could we add a small note about what might require them to look at the details? Something like, "if you update the URL in one of the filters, you may be impacted by this change"?

Reply:
Sure, on it.